### ISSUE TITLE
[Feature Request] Preserve original filenames

### ISSUE THREAD

**Issue by avesst** [akhilrex/podgrab#229]
Hi, 

Thanks for this lovely software, just what I needed. I tried it out and something that stuck out for me was that Podgrab is changing the filenames from their original. I would like to see an option to preserve the original filename. I'm currently using this to backup my own Podcast and would like to keep my filename structure intact.

**Comment by StephenSalomons** [comment]
Yes please. Second the nomination to preserve original filenames.
